k8s/cluster.yaml

```diff
@@ -70,9 +74,16 @@ spec:
       - net.ipv4.tcp_max_orphans
       - net.ipv4.tcp_abort_on_overflow
     streamingConnectionIdleTimeout: 60m
+    registryPullQPS: 3
+    registryBurst: 6
+  kubeControllerManager:
```

Comment:
I'm not 100% convinced these are improving things - we need to run a few clusters with them and decide. They don't seem to be making things worse though.